core/push_consumer.go

```diff
@@ -89,13 +165,24 @@ func newPushConsumer(config *ConsumerConfig) (PushConsumer, error) {
 }
 
 func (c *defaultPushConsumer) Start() error {
-	C.StartPushConsumer(c.cconsumer)
+	code := C.StartPushConsumer(c.cconsumer)
```

Comment:
Maybe functions like this can be move to parent struct？

Reply:
No needed to abstract parent struct, because it simple enough.